src/pytest/test.py

```diff
@@ -248,6 +251,7 @@ def testGet(env):
     # Verify that when a document is deleted, GET returns NULL
     r.cmd('ft.del', 'idx', 'doc10') # But we still keep the document
     r.cmd('ft.del', 'idx', 'doc11')
+    r.cmd('ft.del', 'idx', 'coverage') #should write into log. How is it tested?
```

Comment:
Its not

Reply:
@MeirShpilraien 
`RedisModule_Log(ctx, "warning", "Document %s doesn't exist", RedisModule_StringPtrLen(argv[2], NULL));`

Reply:
I meant you can not verify that log is written